### DISCUSSION TITLE
Lazer replays downloaded from lazer.ppy.sh show score as 0 inside lazer

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#24638] [Q&A] [answered]
Tried on 2 different beatmaps, one is 
https://lazer.ppy.sh/beatmapsets/1856043#osu/3814127
![image](https://github.com/ppy/osu/assets/53081750/e0522a39-cfee-4a0f-936b-e789ec57c2ce)
with this score (https://lazer.ppy.sh/scores/2399877639)
![image](https://github.com/ppy/osu/assets/53081750/dcaea82a-4e98-42ba-8537-29273afa4c60)
Shows like this in game (Acc also changes upon importing?)
![image](https://github.com/ppy/osu/assets/53081750/378e36a7-f9e4-4e7b-982a-cda64a966ec7)

[database.log](https://github.com/ppy/osu/files/12425914/database.log)
[runtime.log](https://github.com/ppy/osu/files/12425915/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24643
